deploy.sh

```diff
@@ -1,4 +1,6 @@
 #!/bin/bash
+
+# run this script as normal user w/o sudo from within the gin-repo root folder.
```

Comment:
Even better then the comment in the source file would be to add checks to make sure what the comment says is true and if not print the comment, i.e. check for the effective user id to be that of the `deploy` user (could be made configurable). The `gin-repo` root folder check is already done in line 13/17.